src/Tools/Microsoft.CodeAnalysis.Toolset.Open/Targets/VSL.Settings.targets

```diff
@@ -79,8 +79,10 @@
   <PropertyGroup Condition="'$(BootstrapBuildPath)' != ''">
     <CscToolPath>$(BootstrapBuildPath)</CscToolPath>
     <CscToolExe>csc.exe</CscToolExe>
+    <CscToolExe Condition="Exists('$(BootstrapBuildPath)\csc2.exe')">csc2.exe</CscToolExe>
```

Comment:
@AlekseyTs are you concerned about cross plat scenarios?  If so both MSBuild and xbuild change directory separators based on the current platform so the hard coding won't be an issue.